ogórek.go

```diff
@@ -238,6 +238,22 @@ func (d Decoder) Decode() (interface{}, error) {
 	return d.pop(), nil
 }
 
+func (d *Decoder) readLine() ([]byte, error) {
+	var has_more bool = true
+	var line []byte = []byte{}
+	for has_more {
+		read_data, is_prefix, err := d.r.ReadLine()
+		if err != nil {
+			return line, err
+		}
+		has_more = is_prefix
+		for i := 0; i < len(read_data); i++ {
+			line = append(line, read_data[i])
```

Comment:
as Damian mentioned, 

`line = append(line, readData...)` and you can remove the loop.